code/datums/autolathe/engineering_vr.dm

```diff
@@ -0,0 +1,22 @@
+// Lets allow autolathes to build the basic tech 1 parts shall we?
```

Comment:
... okay, after it's closed...

SPUDS YOU DUMMEH YOU FORGOT THE AUTOLATHE CIRCUIT BOARD IS STILL NEEDED :V